app/assets/javascripts/components/common/wiki_select.jsx

```diff
@@ -33,21 +33,36 @@ const WikiSelect = createReactClass({
     let wikis = [];
     if (this.props.wikis) {
       wikis = this.props.wikis.map((wiki) => {
+        wiki.language = wiki.language || 'www'; // for multilingual wikis language is null
         return {
-          value: wiki,
+          value: JSON.stringify(wiki),
```

Comment:
What's `stringify` for here?

Reply:
React Select uses the value of value attribute as the key while rendering. Before the value was `wiki` an object, so the key was `wiki.toString()` which is always `[object Object]` so there were multiple collisions. JSON.stringify makes sure the value is a unique string and can also be parsed back to get the object.